rocketpy/rocket/rocket.py

```diff
@@ -138,15 +180,15 @@ def __init__(
         mass : int, float
             Rocket total mass without motor in kg.
         inertia : tuple, list
-            Tuple or list containing the rocket's dry mass inertia tensor
-            components, in kg*m^2.
+            Tuple or list containing the rocket's inertia tensor components,
+            in kg*m^2. This should be measured without motor and propellant.
```

Comment:
The reference for the inertias should be explicitly mentioned to avoid confusion. Also check for other instances of this docstring, for reference see the `Motor` init inertia docstring.

```suggestion
            Tuple or list containing the rocket's inertia tensor components,
            in kg*m^2. This should be measured without motor and propellant
            so that the inertia reference point is the `center_of_mass_without_motor`.
```

Reply:
Good point.

There were no other instance of such description.

Everything fixed now: https://github.com/RocketPy-Team/RocketPy/pull/445/commits/8eb039d332ed4ef71c5d9ef04b537468ddff80b7